how-to/add_controller.rst

```diff
@@ -69,9 +62,9 @@ Deploy controller
 
 2. Now we are ready to bootstrap a controller. Please note the constraints here are the ones used for production JAAS services and should be suitable for most loads. If it is anticipated that the JAAS system will have a different model profile then we encourage you to determine the appropriate constraints for your system: 
 
-    ``juju bootstrap --no-gui --bootstrap-constraints="root-disk=50G cores=8 mem=8G" --config identity-url=$CANDID --config allow-model-access=true --config public-dns-address=$DNS:443 $CLOUD/$REGION $NAME``
+    ``juju bootstrap --bootstrap-constraints="root-disk=50G cores=8 mem=8G" --config login-token-refresh-url=https://$JIMM/.well-known/jwks.json --config allow-model-access=true --config public-dns-address=$DNS:443 $CLOUD/$REGION $NAME``
 
-3. Next the controller should be put into HA mode: 
+3. Next the controller should be put into HA mode (optional): 
 
     ``juju enable-ha``
```

Comment:
Does this rst stuff support markdown like things? I.e., > to put notes?

Reply:
Yeah, there's a primer here https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#rst-primer

Reply:
can't see the > equivalent but if we got it, let's use it for notes